images/gcb-docker-gcloud/cloudbuild.yaml

```diff
@@ -7,8 +7,14 @@ steps:
       - --build-arg=GO_VERSION=$_GO_VERSION
       - .
     dir: images/gcb-docker-gcloud/
```

Comment:
Is this intentional?
I had do use `dir: .` for it work with
```
builder  --project=k8s-staging-test-infra --scratch-bucket=gs://k8s-staging-test-infra-gcb images/gcb-docker-gcloud/
```

Reply:
I think it doesn't need to be this way but may require changes I don't have time to dig into right now. Better to follow the same convention as everything else